bin/broadside

```diff
@@ -166,6 +171,8 @@ on_error do |exception|
     error exception.message, "\nRun your last command with --help for more information."
     false
   else
+    pp exception.message
```

Comment:
adding the env var `GLI_DEBUG=true` will automatically print out the stacktrace, so don't need this